src/components/Accordion/AccordionItem.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <div :class="[toggle ? isActive : '']" @click="accordionhandler($event)">
+    <div :class="[label, 'label_box']">{{ Heading }}</div>
+    <div :class="content" class="content">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+import { ref } from "vue";
+export default {
+  name: "AccordionItem",
+  props: {
+    Heading: String,
+    label: String,
+    content: String,
+    fscontent: {
+      type: String,
+      default: "1.4vw",
+    },
+    fsheading: {
+      type: String,
+      default: "2vw",
+    },
+    Bgheading: {
+      type: String,
+      default: "#2e382b",
```

Comment:
Either give the name of variables in camelCase or PascalCase or snake_case.

Reply:
let's go with camelCase